packages/functions/src/serializer.ts

```diff
@@ -31,8 +31,8 @@ function mapValues(o: object, f: (any) => any): object {
 export class Serializer {
   // Takes data and encodes it in a JSON-friendly way, such that types such as
   // Date are preserved.
-  encode(data: any): any {
-    if (data === null || data === undefined) {
+  encode(data: unknown): unknown {
+    if (data == null) {
```

Comment:
`==` 😱

This seems less readable to me, but if the linter demands it, fine.

Reply:
it's allowed but not demanded. I'm not a fan of it, but we as a team decided to allow it for this specific use case ( Google styleguide also allows it).
I just updated it for the sake of consistency.